### DISCUSSION TITLE
GPS Accuracy

### DISCUSSION THREAD

**Discussion by joewalshwsal** [peterantypas/maiana#110] [Q&A] [answered]
Peter

I have a recent unit purchased in May 22
I very nicely put together and straightforward.

I am testing GPS Accuracy. I have the main unit using a pigtail to an external Antenna with the a 12 foot ethernet cable connecting to the NMEA 0183 which connects to a NMEA2WiFi bridge.
The main board has clear sky access and not enclosed in an enclosure. 
I am monitoring using Navionics and have set in tracking. I see the track jumps around by as much as 50 feet.

See attached;

![FWB](https://user-images.githubusercontent.com/106410868/171607576-62c9b0c8-cb15-4174-9ce6-c183b6a5a2cb.jpg)

Is this considered normal? 

I have a small boat and was hoping this unit would act as the principal GPS. 
I use Navionics as a navigation tool and by means of the NMEA2WiFi, Navionics will pick up the AIS unit GPS.
My sailing area is within a busy shipping area so AIS is important to me.
I do have a GPS on my VHF radio but if I am correct the AIS unit cannot take in a NEMA stream to provide GPS into the AIS ?

An alternative might be;
If the AIS unit did not send GPS based NMEA sentences out, the Navionics would use the tablet GPS. Is this possible ?
Really all I need Navionics to pick up is AIS sentences 

thanks in advance 

JoeW

**Comment by peterantypas** [comment]
The GPS antenna is tuned for the enclosure, not for free space. Please assemble it completely (you don't have to melt the heat shrink though) and test again.

**Reply by peterantypas** [reply]
See #82

**Reply by joewalshwsal** [reply]
Peter

I will do this now and test

ta

**Comment by joewalshwsal** [comment, marked as answer]
Peter

A picture paints 1000 words see below
You are 100% correct with your enclosure the fix is rock solid

![fix](https://user-images.githubusercontent.com/106410868/171662784-07cee1fb-e9d0-43d0-8f60-4062a776c918.jpg)
